.github/workflows/CI.yaml

```diff
@@ -49,6 +49,9 @@ jobs:
           distribution: temurin
           java-version: 17
           cache: gradle
+      - name: install dependencies
+        run: sudo apt install jq
+        if: always()
```

Comment:
According to https://github.com/actions/runner-images/blob/ubuntu22/20230217.1/images/linux/Ubuntu2204-Readme.md
jq version 1.6 is pre-installed in the runner.

Reply:
Thanks for the hint. Done. I only checked if an ubuntu on our dev servers has jq installed, which they do not. So I did not expect it on the image.

Reply:
ok